src/zui/ZUIEditTextInPlace/index.tsx

```diff
@@ -143,15 +176,39 @@ const ZUIEditTextinPlace: React.FunctionComponent<ZUIEditTextinPlaceProps> = ({
     }
   };
 
+  const valueText = () => {
```

Comment:
What is the purpose of all of this logic? To me it seems like it's overkill, and just using `value={editing? text : (text || placeholder)}` seems to work for me, but I might be missing some edge case that you've bumped into?

Reply:
I misunderstood the design, so I thought when user types the event, it would show the text as placeholder in title. So the first screenshot is what I made, and second one is what you suggest.
![image](https://user-images.githubusercontent.com/77925373/232013002-558caea2-82e0-4f0d-8931-be19a78e97ec.png)